README.md

```diff
@@ -270,8 +270,7 @@ N.B. by default PNG and JPEG are not modified in the middleware to prevent break
 To enable them, there are configuration options for each image type as well as an option to disable the PDF middleware
 (on by default).
 
-If either of the image handling middleware options are enabled, the [ignore_path](#ignore_path) should also
-be configured, otherwise assets are likely to be handled which would likely result in 404 responses.  
+If either of the image handling middleware options are enabled, the [ignore_path](#ignore_path) and/or [ignore_request](#ignore_request) should also be configured, otherwise assets are likely to be handled which would likely result in 404 responses.  
```

Comment:
Can you please split these over multiple lines. Although it doesn't make a difference for the rendered markdown, it's easier to read in most editors if the line length is kept under 120.

```suggestion
If either of the image handling middleware options are enabled, the [ignore_path](#ignore_path) and/or
[ignore_request](#ignore_request) should also be configured, otherwise assets are likely to be handled
which would likely result in 404 responses.  
```